test/dlc_tests/ecs/pytorch/training/test_ecs_pytorch_training.py

```diff
@@ -93,6 +96,11 @@ def test_ecs_pytorch_training_dgl_gpu(gpu_only, py3_only, ecs_container_instance
     Given above parameters, registers a task with family named after this test, runs the task, and waits for
     the task to be stopped before doing teardown operations of instance and cluster.
     """
+    _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
+    image_cuda_version = get_cuda_version_from_tag(pytorch_training)
+    if Version(image_framework_version) == Version("1.6") and image_cuda_version == "cu110":
+        pytest.skip("DGL does not suport CUDA 11 for PyTorch 1.6")
```

Comment:
Does this test still need to be skipped? DGL has been installed using a custom-built wheel, right?

Reply:
Even with custom build this issue is noted